app/src/main/java/ch/epfl/sdp/drone3d/service/impl/mission/ParallelogramMappingMissionService.kt

```diff
@@ -5,6 +5,7 @@
 
 package ch.epfl.sdp.drone3d.service.impl.mission
 
+import android.util.Log
```

Comment:
Log should not be used, please use Timber

Reply:
Actually, the import was unused, I forgot to remove it